scala-package/mxnet-demo/java-demo/README.md

```diff
@@ -71,9 +73,15 @@ If you want to test run on GPU, you can set a environment variable as follows:
 export SCALA_TEST_ON_GPU=1
 ```
 ## Clean up
-Clean up for Maven package is simple, you can run the pre-configed `Makefile` as:
+Clean up for Maven package is simple:
 ```Bash
-make javaclean
+mvn clean
+```
+
+## Run in Eclipse
+You can convert the maven project to the eclipse one by running
+```
+mvn eclipse:eclipse
```

Comment:
This is a repeat of before... why is this here?
Also this is a conversion instruction not a "run" instruction as the title suggests. Maybe you can include some screenshots of how you open the project in eclipse and build it, etc...

Reply:
This is specifically for users trying to use Eclipse in their demo project